hosted-zone-api/src/main/java/ai/vespa/cloud/SystemInfo.java

```diff
@@ -42,8 +46,8 @@ public Cloud cloud() {
         return cloud;
     }
 
-    /** Returns the cluster this is part of */
-    public Cluster cluster() { return cluster; }
```

Comment:
This is used by stream-ranking:
https://git.ouryahoo.com/streambe/massmedia_serving/blob/master/homerun/src/main/java/com/yahoo/jarvis/docproc/CleanupExpiredGlobalscoresDocProc.java#L84